ts/Stock/Indicators/APO/APOIndicator.ts

```diff
@@ -0,0 +1,242 @@
+/* *
+ *
+ *  License: www.highcharts.com/license
+ *
+ *  !!!!!!! SOURCE GETS TRANSPILED BY TYPESCRIPT. EDIT TS FILE ONLY. !!!!!!!
+ *
+ * */
+
+'use strict';
+
+const {
+    seriesTypes: {
+        ema: EMAIndicator
+    }
+} = BaseSeries;
+import type {
+    APOOptions,
+    APOParamsOptions
+} from './APOOptions';
+import type APOPoint from './APOPoint';
+import type IndicatorValuesObject from '../IndicatorValuesObject';
+import type LineSeries from '../../../Series/Line/LineSeries';
+import BaseSeries from '../../../Core/Series/Series.js';
+import RequiredIndicatorMixin from '../../../Mixins/IndicatorRequired.js';
+import U from '../../../Core/Utilities.js';
+const {
+    extend,
+    merge,
+    error
+} = U;
+
+const EMA = BaseSeries.seriesTypes.ema;
```

Comment:
Replace (delete) the usage of the `EMA` constant in favor of the deconstructed `EMAIndicator`.